### ISSUE TITLE
Offscreen text for Image Copyright

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#483]
**Is your feature request related to a problem? Please describe.**
We should apply offscreen text for the copyright of an image for screen-readers

**Describe the solution you'd like**
This offscreen text should be `Copyright  Getty  Images.`, where Getty Images is the value from the copyright field in the data feed. The screen-reader order should be `image -> copyright -> caption`.

We should use the visually hidden component.

**Testing**
- Use a screenreader to ensure that the copyright is read out, and is in the correct order
- Cypress test

**Additional Context**
- https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology
